config/routes/test/sylius_test_plugin.yaml

```diff
@@ -1,5 +1,5 @@
 sylius_test_plugin_main:
     path: /test/main
-    controller: FrameworkBundle:Template:template
+    controller: Symfony\Bundle\FrameworkBundle\Controller\TemplateController::templateAction
```

Comment:
Should this change be mentioned in UPGRADE file? Or maybe it is mentioned somewhere, but I've missed that?
I've checked also the upgrade for [SyliusThemeBundle](https://github.com/Sylius/SyliusThemeBundle/blob/master/UPGRADE.md)

Reply:
I guess we don't need to mention that, since it's just an exemplary code - however, we should make sure Sylius-Standard for new projects based on 1.9 has this changed.